packages/semantic-ui/src/TextWidget/TextWidget.js

```diff
@@ -1,48 +1,57 @@
 /* eslint-disable react/prop-types */
 import React from "react";
-import PropTypes from "prop-types";
 import { Form } from "semantic-ui-react";
 import { getSemanticProps } from "../util";
-import { utils } from "@rjsf/core";
+import {  utils } from "@rjsf/core";
 const { getDisplayLabel } = utils;
 function TextWidget(props) {
   const {
     id,
     placeholder,
-    required,
-    readonly,
-    disabled,
     name,
     label,
-    schema,
-    uiSchema,
     value,
+    required,
+    readonly,
+    disabled,
     onChange,
     onBlur,
     onFocus,
     autofocus,
     options,
+    schema,
+    uiSchema,
     formContext,
   } = props;
-  const semanticProps = getSemanticProps({ formContext, options });
+  const semanticProps = getSemanticProps(
+    { formContext, options,
+      uiSchema,
+      defaultSchemaProps: {
+        fluid: true,
+        inverted: false,
+    }
+  });
   // eslint-disable-next-line no-shadow
   const _onChange = ({ target: { value } }) =>
     onChange(value === "" ? options.emptyValue : value);
   const _onBlur = () => onBlur && onBlur(id, value);
   const _onFocus = () => onFocus && onFocus(id, value);
-  const inputType = schema.type === 'string' ?  'text' : `${schema.type}`;
+
   const displayLabel = getDisplayLabel(
     schema,
     uiSchema
     /* TODO: , rootSchema */
   );
+
+  const inputType = schema.type === 'string' ?  'text' : `${schema.type}`;
```

Comment:
You could just revert these changes since the location of `inputType` in the code does not matter